File: Sorting/BubbleSort/bubbleSort.java
package DSA.Sorting.BubbleSort;

import java.util.*;

class bubbleSort 
{
    static void sort(int arr[]) 
    {
        int size = arr.length;
        for (int i = 0; i < size - 1; i++) 
        {
            int flag=0;
            for (int j = 0; j < size - i - 1; j++) 
            {
                if (arr[j] > arr[j + 1]) 
                {
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                    flag =1;
                }
            }
            if(flag == 0)
            {
                break;
            }
        }
    }
    
    public static void main(String[] args) {
        int[] data = {-43, 43, 42, 23, 4, 2, -43, 0};
        sort(data);
        System.out.println(Arrays.toString(data));  // Use Arrays.toString() to print the array
    }
}

File: Sorting/BubbleSort/bubbleSort.py
def BubbleSort(data):
    for i in range(len(data)):
        flag=0
        for j in range(0,len(data)-i-1):
            if data[j] > data[j+1]:
                #temp = data[j]
                #data[j] = data[j+1]
                #data[j+1] = temp
                data[j] , data[j+1] = data[j+1] , data[j]
                flag=1
        if(flag==0):
            break;
                
data = [-4,4,23,0,-34,534,-43333]
BubbleSort(data)
print(data)